documentation/src/mkdocs.yml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+site_name: 'Exploring Pravega'
+site_url: 'https://pravega.io/'
+site_description: 'Documentation to help you get familiar with Pravega'
+
+repo_url: 'https://github.com/pravega/schema-registry'
+edit_uri: 'tree/master/documentation/src/docs'
+
+theme: 'material'
+
+
+# Extensions
+markdown_extensions:
+  - admonition
+  - codehilite(guess_lang=false)
+  - footnotes
+  - meta
+  - toc(permalink=true)
+  - pymdownx.betterem(smart_enable=all)
+  - pymdownx.caret
+  - pymdownx.inlinehilite
+  - pymdownx.magiclink
+  - pymdownx.smartsymbols
+  - pymdownx.superfences
+
+pages:
+  - Overview: 'overview.md'
+  - Understanding Schema Registry:
+      - 'Schema Registry Concepts': 'schemaregistry-concepts.md'
+
+  - Using Schema Registry SDK:
+      - 'Java API Reference': 'javadoc.md'
```

Comment:
`javadoc.md` is missing?

Reply:
i wanted to check with you on this - should i copy over javadocs generated from javadocs task in the documents or are they picked from elsewhere?